src/main/java/com/kelinci/mainapp/api/HomeController.java

```diff
@@ -1,13 +1,11 @@
 package com.kelinci.mainapp.api;
 
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Raczej staramy się w projekcie nie używać importu *, ponieważ to zaciemnia informację, których rzeczy tak naprawdę używamy

Reply:
ok, poprawię.